src/databricks/labs/ucx/hive_metastore/locations.py

```diff
@@ -69,7 +69,8 @@ def location(self) -> str | None:
     def _parse_location(cls, location: str | None) -> list[str]:
         if not location:
             return []
-        parse_result = cls._parse_url(location.rstrip("/"))
+        location = ExternalLocations.clean_location(location)
```

Comment:
This looks duplicate, the location goes into `clean_location` twice

Reply:
Fixed